docs/guide/upgrade/README.md

```diff
@@ -1,4 +1,111 @@
-# Upgrade from v3 to v4
+# Migration guides
+
+## Upgrade from v4 to v5
```

Comment:
Add:

- If using the `ssr: true` mode, rename `ctx.res.verifiedFireAuthUser` to `ctx.res.locals.user` in your **nuxtServerInit**.

Maybe add a comment in general saying:

- We have completely changed how SSR Authentication works. If you have set up SSR like previously mentioned [here](http://localhost:8080/tutorials/ssr), follow the Tutorial again to properly set it up with v5.

Reply:
Done (please check)